Source/Core/DolphinQt/NetPlay/NetPlayBrowser.h

```diff
@@ -35,6 +35,8 @@ class NetPlayBrowser : public QDialog
   void accept() override;
 signals:
   void Join();
+  void UpdateStatusRequested(const QString& status);
+  void UpdateListRequested(const std::vector<NetPlaySession>& sessions);
```

Comment:
```suggestion
  void UpdateListRequested(std::vector<NetPlaySession> sessions);
```

or

```suggestion
  void UpdateListRequested(std::vector<NetPlaySession>&& sessions);
```

Otherwise moving into emissions of this signal won't actually perform a move.

Reply:
That makes sense. I should have added the `const&`.

I'll update it.

Reply:
Done.